sound/soc/codecs/rt715-sdca-sdw.c

```diff
@@ -230,6 +230,8 @@ static int __maybe_unused rt715_dev_resume(struct device *dev)
 	if (!slave->unattach_request)
 		goto regmap_sync;
 
+	slave->unattach_request = 0;
```

Comment:
we need mutex here or make sure the sequence of system resume and runtime pm resume

Reply:
> we need mutex here or make sure the sequence of system resume and runtime pm resume

Yes, a mutex is needed. Thanks @RanderWang